packages/edit-site/src/components/global-styles/font-library-modal/resolvers.js

```diff
@@ -7,13 +7,54 @@
  */
 import apiFetch from '@wordpress/api-fetch';
 
-export async function fetchInstallFont( data ) {
+export async function fetchInstallFontFamily( data ) {
 	const config = {
 		path: '/wp/v2/font-families',
 		method: 'POST',
 		body: data,
 	};
-	return apiFetch( config );
+	return apiFetch( config ).then( ( response ) => {
+		return {
+			id: response.id,
+			...response.font_face_settings,
+			fontFace: [],
+		};
+	} );
```

Comment:
```suggestion
	const response = await apiFetch( config );
	return {
		id: response.id,
		...response.font_face_settings,
		fontFace: [],
	};
	} );
```

Either this should await or this function should not be async.

Reply:
Thanks! Looks like we have a mix of `async` and `then`. Addressed in this separate PR here: https://github.com/WordPress/gutenberg/pull/57961/commits/1d01919c0563e94f490c8e0d053b6db4f763f113.